client/my-sites/checklist/main.jsx

```diff
@@ -0,0 +1,171 @@
+/**
+ * External dependencies
+ *
+ * @format
+ */
+import React, { Fragment, PureComponent } from 'react';
+import { connect } from 'react-redux';
+import { localize } from 'i18n-calypso';
+import { find } from 'lodash';
+
+/**
+ * Internal dependencies
+ */
+import config from 'config';
+import EmptyContent from 'components/empty-content';
+import FormattedHeader from 'components/formatted-header';
+import ChecklistShow from './checklist-show';
+import Main from 'components/main';
+import DocumentHead from 'components/data/document-head';
+import { requestSiteChecklistTaskUpdate } from 'state/checklist/actions';
+import { getSelectedSiteId } from 'state/ui/selectors';
+import isSiteAutomatedTransfer from 'state/selectors/is-site-automated-transfer';
+import { isJetpackSite, getSiteSlug } from 'state/sites/selectors';
+import { getCurrentUser } from 'state/current-user/selectors';
+import QuerySiteChecklist from 'components/data/query-site-checklist';
+import { recordTracksEvent } from 'state/analytics/actions';
+import { createNotice } from 'state/notices/actions';
+import { requestGuidedTour } from 'state/ui/guided-tours/actions';
+import ChecklistShowShare from './share';
+import SidebarNavigation from 'my-sites/sidebar-navigation';
+import PageViewTracker from 'lib/analytics/page-view-tracker';
+
+class ChecklistMain extends PureComponent {
+	getHeaderTitle( displayMode ) {
+		if ( displayMode === 'free' ) {
+			return 'Your site has been created!';
```

Comment:
I was surprised to see that some of the strings here are localized, while others are not.